documentation/docs/steps/handlePipelineStepErrors.md

```diff
@@ -18,7 +21,7 @@ none
 * `echoDetails` - If set to true the following will be output to the console:
     1. Step beginning: `--- BEGIN LIBRARY STEP: ${stepName}.groovy ---`
     2. Step end: `--- END LIBRARY STEP: ${stepName}.groovy ---`
-    3. Step errors: 
```

Comment:
[nit] not covered by title of pull request

Reply:
autofix of my editor 😄